### PR TITLE
feat: Send browser notification when chat closed.

### LINKED ISSUE #476: [FEATURE] send a notification when a user leaves a chat
### Description

Send a browser notification when one of the user leaves the chat, there's already a pop-up when that happens, but if a user isn't on that page then the user can miss it. 
https://github.com/Dun-sin/Whisper/blob/4b12bf53561e7c2e5f8188c40c1a7d10eba11978/client/src/components/BuddyMatcher.jsx#L91-L106

### Additional Information
there's already a function for notification, just call it at the right position

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by kapooraryan**
# Fixes Issue

Closes #476 

# 👨‍💻 Changes proposed(What did you do ?)

Called the function createBrowserNotification to send a browser notification when one of the user leaves the chat but isn't on the page.

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

I have not incluced the requestBrowserNotificationPermissions function here but do let me know if I should, or if not the code should just work fine.

# 📷 Screenshots

NA

### CODE REVIEW

**Comment by Dun-sin** [comment]
@kapooraryan use a better title, don't be lazy

**Comment by kapooraryan** [comment]
> @kapooraryan use a better title, don't be lazy

Hey, @Dun-sin it's updated now.

**Comment by Dun-sin** [comment]
> > @kapooraryan use a better title, don't be lazy
> 
> Hey, @Dun-sin it's updated now.
> 
> 

A summary pls😭

**Comment by Dun-sin** [comment]
@kapooraryan look at other PRs and learn how to make a better PR title

**Comment by kapooraryan** [comment]
> @kapooraryan look at other PRs and learn how to make a better PR title

Is it better now @Dun-sin 😅?

**Comment by Dun-sin** [comment]
> > @kapooraryan look at other PRs and learn how to make a better PR title
> 
> Is it better now @Dun-sin 😅?

Summary, if you can't put it in less than 6-8 words it's not a summary

**Comment by kapooraryan** [comment]
> No, a Summary, if you can't put it in less than 6-8 words it's not a summary or a good PR title, also when you checked other PRs did you see `fixes <whatever>`

No, I did not. Hope I have got it right.

File: client/src/components/BuddyMatcher.jsx
Comment (Dun-sin) [inline client/src/components/BuddyMatcher.jsx:1]: remove all format changes here

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/BuddyMatcher.jsx
Comment (kapooraryan) [reply client/src/components/BuddyMatcher.jsx:1]: Done as per requested format @Dun-sin .

File: client/src/components/BuddyMatcher.jsx [hunk 1]
```diff
@@ -121,6 +121,7 @@ const BuddyMatcher = () => {
 				return;
 			}
 
+			createBrowserNotification('Chat Closed', 'This chat is closed! Would you like to search for a new buddy?');
```
Comment (Dun-sin) [inline client/src/components/BuddyMatcher.jsx:124]:
```suggestion
			createBrowserNotification('Chat Closed', 'Your buddy left the chat');
```

**Pull request merged**